pkg/hooks/hook_env.go

```diff
@@ -74,12 +74,12 @@ func (h *HookEnv) PodFromDisk() (*pods.Pod, error) {
 		return nil, err
 	}
 
-	return pods.PodFromPodHome(types.NodeName(node), podHome)
+	return pods.PodFromPodHome(types.NodeName(node), podHome, "")
```

Comment:
This pattern is coming up a lot. Consider creating a new constructor that doesn't require the caller to know the suitable default value. When we want a pod factor with a require file, we can use this 3 argument constructor.